pages/profile/[userId].tsx

```diff
@@ -230,6 +233,15 @@ export default function InternalProfile({
         <AchievementComponent userId={userId} />
       ),
     },
+    {
+      shouldShow:  userId == "R7nzMKiRewgJuLm54eQ1KdHV3g82",
+      title: `Interview Tracking`,
+      hasProgress: true,
+      value: (userBadgeCount * 100) / totalBadgeCount,
+      component: typeof userId == "string" && (
+        <OfferTable  isCAD={isCAD} year={year} data={LuckyInterviewData}/>
```

Comment:
Instead of calling your hardcoded data here, create a const like I suggested earlier and use that here. It could potentially be null if the user id doesn't match

Reply:
will do, sounds good